frontend/app/src/scenes/Home/scenes/AddCategory/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {
+  Button,
+  Col,
+  Form,
+  Input,
+  notification,
+  Row,
+  Spin,
+  Typography,
+} from 'antd';
+import mainStyles from '../../styles.css';
+import { AddCategoryStateProps } from './interfaces';
+import { RouteComponentProps, withRouter } from 'react-router';
+import axios, { AxiosResponse } from 'axios';
+import { Category } from '../../interfaces';
+import { handleApiError } from '../../../../services/util/errorHandler';
+
+const { Title, Text } = Typography;
+
+class AddCategory extends React.Component<
+  RouteComponentProps,
+  AddCategoryStateProps
+> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      isLoading: false,
+    };
+  }
+
+  onFinish = (values: any) => {
+    this.setState({ isLoading: true });
+    axios
+      .post(`${window.origin}/core/academix/admin/categories`, values)
+      .then((res: AxiosResponse<Category>) => {
+        if (res.status == 201) {
+          this.setState({ isLoading: false });
+          notification.success({
+            message: 'Success!',
+            description: 'Successfully Created a Category',
+          });
+          this.props.history.push('/dashboard/academix/categories');
```

Comment:
Let's redirect to the created category's page. It's hard to identify if there are many categories. (when paginating)